input/LimitStateDesign.tex

```diff
@@ -527,7 +552,7 @@ \section{Unified Approach}
 But $$n=n_{max} \text{ (assumed constant for ductility)}$$
 
 This is a region when the value of X is not ﬁxed.For a ﬁxed arrangement
-of steelin section,this region reduces to a straight line. \\
+of steelin section,this region reduces to a straight line. 
 
 Region 4. Section partly under compression and partly under tension,design
```

Comment:
**Line 554**: "fixed.For".

**Line 555**:
1. "steelin"
2. "section,this"

**Line 557**:
"tension,design"